### PR TITLE
Fix private repo deploy errors

### PR DESCRIPTION
**Pull request by bobbyiliev**

### CODE REVIEW

**Review by littlewish89 — COMMENTED** [review]
Command

File: deploy.json [hunk 1]
```diff
@@ -199,9 +200,10 @@
...
                 "name": "migrate",
-                "git": {
-                    "repo_clone_url": "https://github.com/${wave.repo}.git",
-                    "branch": "main"
+                "github": {
+                    "repo": "${wave.repo}",
+                    "branch": "main",
+                    "deploy_on_push": true
                 },
                 "run_command": "php artisan migrate --seed ; php artisan storage:link",
```
[the remark this answers is not in the document]
Comment (bobbyiliev) [reply deploy.json:206]: Hi there,

 Not 100% sure what you mean by Ruby. Can you clarify please?

File: deploy.json [hunk 2]
```diff
@@ -4,9 +4,10 @@
         "services": [
             {
                 "name": "web",
```
Comment (littlewish89) [inline deploy.json:6]: Password

File: deploy.json (hunk 1 above)
Comment (littlewish89) [inline deploy.json:206]: Run

File: .do/app.yaml [hunk 3]
```diff
@@ -1,9 +1,10 @@
 name: wave
```
[the remark this answers is not in the document]
Comment (littlewish89) [reply .do/app.yaml:1]: Run

**Pull request merged**